main.tf

```diff
@@ -1,10 +1,4 @@
 terraform {
-  cloud {
-    organization = "liquibase"
-    workspaces {
-      name = "liquibase-github-actions"
```

Comment:
Is removing this correct? How will spacelift know which org should own all the repos?

Reply:
Did it get moved to the generate.yaml workflow? I see some spacectl commands with `--id` but don't know if that will do what this line used to do.

Reply:
@mcred the `cloud` block needs to be removed. If not, Spacelift will fail. Check https://spacelift.io/blog/how-to-migrate-from-terraform-cloud (**Post-Migration Cleanup** section).

Spacelift knows which org to use thanks to the following vars:

```
          SPACELIFT_API_KEY_ENDPOINT: ${{ secrets.SPACELIFT_API_KEY_ENDPOINT }}
          SPACELIFT_API_KEY_ID: ${{ secrets.SPACELIFT_API_KEY_ID }}
          SPACELIFT_API_KEY_SECRET: ${{ secrets.SPACELIFT_API_KEY_SECRET }}
```

And it knows the **workspace** to use (**stack** - Spacelift terminology) by using the `--id` argument.